src/js/common/browserInfoHelper.js

```diff
@@ -78,20 +90,59 @@ const getFormattedOffset = () => {
     return formattedUTC;
 }
 
+/**
+ * The function returns users timezone offset relative to UTC
+ * @returns {string} UTC concatenated with users broesers offset 
+ */
 export const getTimezone = () => `UTC${getFormattedOffset()}`;
+
+/**
+ * The function returns validated screens width value
```

Comment:
I think these comments should be:

`Function that validates {the user's screen's width value}, and then returns it. If it fails validation, it returns null`